paasta_tools/cli/utils.py

```diff
@@ -968,6 +971,7 @@ def get_instance_configs_for_service(
                     cluster=cluster,
                     soa_dir=soa_dir,
                     load_deployments=False,
+                    for_validation=False,
```

Comment:
we should be fine leaving this call alone - `False` is already the default

Reply:
(we really just need to pass down `for_validation` where necessary)

Reply:
(from chatting in gmeet: i think that this is mypy not understanding that this value has a default based on how we type the function signature)

Reply:
we can fix this (and the load_deployments one) by using `DefaultNamedArg` instead of `NamedArg` - but we/I can do that in a follow-up to not extra changes to this PR